lib/cfd_trading/cfd_order_confirmation.dart

```diff
@@ -73,28 +73,30 @@ class CfdOrderConfirmation extends StatelessWidget {
                         children: [
                           ElevatedButton(
                               onPressed: () async {
-                                // TODO: Plug in actual state changes
-
                                 try {
-                                  // TODO: Don't hardcode the taker amount
                                   await api.openCfd(order: order);
                                   FLog.info(text: 'OpenCfd returned successfully');
 
+                                  // refreshing cfd list after cfd has been opened
+                                  api.listCfds();
```

Comment:
❓ This has a side-effect? We're not doing anything with the returned value.

Reply:
It does. The name is a bit confusing. Something like `syncCfds` or `updateCfds` might be clearer.

Reply:
I went for `refreshCfds` and also fixed the commit, because here we were only calling `api.listCfds()`, which returns the list which we were ignoring. Instead we had to call:

```dart
await cfdTradingService.refreshCfdList();
```

Reply:
Fixed in fd57d7499d73a3cac748ec27fb0ee825ae0d998c.